main.js

```diff
@@ -1,8 +1,23 @@
-const KEY_CODE_SPACE = 32;
+var jsPsych = initJsPsych({
```

Comment:
I know that the authors of jsPsych use `var` all the time (it's in the migration guide), however, throughout our code base we generally prefer to use `let` instead. It probably doesn't make a difference here.